src/_css/AddAssetComponent.css

```diff
@@ -0,0 +1,8 @@
+.form-wrapper {
+    width: 70%;
+}
+
+.label-style {
+    width: 100px !important;
```

Comment:
Use of `!important` is not encouraged. Why are you using it here? Read more on [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) to determine if you really need it